experimental/packages/opentelemetry-exporter-prometheus/test/PrometheusExporter.test.ts

```diff
@@ -211,7 +211,7 @@ describe('PrometheusExporter', () => {
       const deferred = new Promise<void>(res => {
         resolve = res;
       });
-      mockResponse.end.callsFake(() => resolve());
+      mockResponse.end.callsFake(() => resolve() as any);
```

Comment:
Required because of updated types. The `callsFake` expects the callback to return a reponse object but it returns void. Since it isn't used there isn't any requirement to mock the response.